### PR TITLE
(Partner ID: cyware-238) README.md and increasing the auth token timeouts. Added No IOC Found message when the IOCs is not found in CTIX

### PR DESCRIPTION
**Pull request by gladipero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CTIX/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Cyware Threat Intelligence eXchange
+- Added suitable when IOC is not found in CTIX.
```
Comment (yuvalbenshalom) [inline Packs/CTIX/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added handling to the case where the input IOC is not found in CTIX.
```

**Comment by DeanArbel** [comment]
@yuvalbenshalom @gladipero This PR has become a bit stale. What's the status here?

**Comment by gladipero** [comment]
@DeanArbel Might need help from someone from the XSOAR side in resolving the conflicts. Tried multiple times to rebase and push again from my end but unable to resolve the conflicts. 
Only other way I can think of now is to create a new PR altogether.

**Comment by DeanArbel** [comment]
> @DeanArbel Might need help from someone from the XSOAR side in resolving the conflicts. Tried multiple times to rebase and push again from my end but unable to resolve the conflicts. 
> Only other way I can think of now is to create a new PR altogether. 

@gladipero I can assist solving the conflicts, however it might be indeed better to start a new page. Please open a new PR, and close this one once you've done so.

**Comment by gladipero** [comment]
Closing this, created a new https://github.com/demisto/content/pull/17142/

**Pull request closed without merging**